SkimsAUX/plugins/prodJets.cc

```diff
@@ -133,9 +143,13 @@ prodJets::prodJets(const edm::ParameterSet & iConfig)
 
   produces<std::vector<int> >("trksForIsoVetoMatchedJetIdx");
   produces<std::vector<int> >("looseisoTrksMatchedJetIdx");
+  //PUPPI
+  produces<std::vector<TLorentzVector> >("puppiJetsLVec");
+  //ak8
+  produces<std::vector<TLorentzVector> >("ak8JetsLVec"); 
+  produces<std::vector<int> >("qgToken");
```

Comment:
should be std::vector<double> instead of std::vector<int>? And the qgToken is not produced (no iEvent.put found for it)